.circleci/config.yml

```diff
@@ -50,15 +50,15 @@ jobs:
       - image: node:8
         user: node
     <<: *unit_tests
-  node9:
+  node10:
     docker:
-      - image: node:9
+      - image: node:10
         user: node
     <<: *unit_tests
 
   publish_npm:
     docker:
-      - image: node:8
+      - image: node:10
```

Comment:
It wouldn't matter much but isn't it still better to publish with 8 because it's LTS?

Reply:
Modules published with 10, should work with other versions of Node too, so is there a tangible difference here?

Reply:
Not really. LGTM.